waf_unsupported_target.go

```diff
@@ -4,14 +4,15 @@
 // Copyright 2016-present Datadog, Inc.
 
 // Unsupported target OS or architecture on a supported Go version
-//            Unsupported OS        Unsupported Arch      Good Go Version
-//go:build ((!linux && !darwin) || (!amd64 && !arm64)) && !go1.22
+//            Unsupported OS        Unsupported Arch   Good Go Version Not manually disabled
+//go:build ((!linux && !darwin) || (!amd64 && !arm64)) && !go1.22 && !datadog.no_waf
```

Comment:
Don't you need to relax this build condition to cumulate multiple `wafSupportErrors` cases?
To only target unsupported targets:
 ```suggestion
//go:build ((!linux && !darwin) || (!amd64 && !arm64))
```

Reply:
Same comment with the other files appending to this var.

Reply:
Same comment with the other files appending to this var.

Reply:
right